frontend/src/component/pages/shared/CreateShortLinkSection.tsx

```diff
@@ -59,6 +61,12 @@ export class CreateShortLinkSection extends Component<Props> {
           </div>
         </div>
         <div className={'input-error'}>{this.props.inputErr}</div>
+        <div className={'public-listing-toggle'}>
```

Comment:
I rethought about this. If we change the full border to a top divider ( border-top ), it may look cooler! Because the backend is not ready yet, we should use feature flag to turn it off ( hide it from homepage ) and turn it back on when it's ready.

You can move `<div className={'public-listing-toggle'}>` to [here](https://github.com/short-d/short/blob/master/frontend/src/component/UIFactory.tsx) and create a static feature flag [here](https://github.com/short-d/short/blob/master/frontend/src/service/StaticConfigDecision.service.ts)

Reply:
TODO
- [ ] Move `public-listing-toggle` element to `UIFactory`
- [ ] Change border to just border-top
- [ ] Add feature flag to hide the public listing toggle from homepage